docs/pai-management/doc/customized-configuration.md

```diff
@@ -0,0 +1,54 @@
+<!--
+  Copyright (c) Microsoft Corporation
+  All rights reserved.
+
+  MIT License
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+  documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+  the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+  to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+  BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+
+# OpenPAI Configuration Document
+
+
+
+### Index
+
+- [Overview](#overview)
+- [Step a. How to write cluster-configuraiton.yaml](#step_a)
+- [Step b. How to write kubernetes-configuration.yaml](#step_b)
+- [Step c. How to write service-configuraiton.yaml](#step_c)
+
+
+#### Overview <a name="overview"></a>
+
+PAI configuration consists of 4 YAML files:
+
+- [`cluster-configuration.yaml`](./how-to-write-pai-configuration.md#cluster_configuration) - Machine-level configurations. This file contains basic configurations of cluster, such as the login info, machine SKUs, labels of each machine, etc.
+- [`kubernetes-configuration.yaml`](./how-to-write-pai-configuration.md#kubernetes_configuration) - Kubernetes-level configurations. This file contains basic configurations of Kubernetes, such as the version info, network configurations, etc.
+- [`k8s-role-definition.yaml`](./how-to-write-pai-configuration.md#k8s_role_definition) - Kubernetes-level configurations. This file contains the mappings of Kubernetes roles and machine labels. It will be deprecated in the future.
+- [`serivices-configuration.yaml`](./how-to-write-pai-configuration.md#services_configuration) - Service-level configurations. This file contains the definitions of cluster id, docker registry, and those of all individual PAI services.
+
+### Step a. How to write cluster-configuration.yaml  <a name="step_a"></a>
+
+[A Guide of customize cluster-configuration.yaml](./how-to-configure-cluster-configuraiton.md)
+
+### Step b. How to write kubernetes-configuration.yaml <a name="step_b"></a>
+
+[A Guide of customize kubernetes-configuration.yaml](./how-to-configure-k8s-config.md)
+
+### Step c. How to write service-configuration.yaml <a name="step_c"></a>
+
+[A Guide of customize service-configuration.yaml](./how-to-congiure-service-config.md)
+
```

Comment:
Question: Lack k8s-role-definition.yaml or there is no need to config this?

Reply:
Yes